samples/excel/default.yaml

```diff
@@ -11,8 +11,9 @@ script:
 
         async function run() {
             await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getActiveWorksheet();
 
-                OfficeHelpers.UI.notify("Your code goes here");
+                console.log("Your code goes here...");
```

Comment:
Why the "..." here but not in the others?
Oh, I see, that was what I actually had copy-pasted for you.  My bad.  I didn't want the "..."s there.  I suggest we remove them.